docs/reference/changelog.md

```diff
@@ -0,0 +1,23 @@
+# Changelog
+
+## [1.0.1] - yyyy-mm-dd
```

Comment:
I think it's just plain 1.0.0, since the last release was 1.0.0.beta (otherwise, we're skipping 1.0.0). Another option is 1.0.0.beta2 if we don't want 1.0.0 yet.

Reply:
I've gone with beta2, I'm still somewhat uncomfortable declaring it 1.0.0 until we've had more examples of implementation